src/http_server/api/v1/metrics.c

```diff
@@ -275,20 +278,28 @@ void cb_metrics_prometheus(mk_request_t *request, void *data)
                                                  + mk.via.str.size
                                                  + sk.via.str.size
                                                  + len + time_len + 28);
-                if (sds_metric == NULL) {
-                    goto error;
-                }
+                sds_null_check(sds_metric);
 
                 sds_metric = flb_sds_cat(sds_metric, "fluentbit_", 10);
```

Comment:
a cosmetic change, would you please use a temporal variable and do a re-assignation in the macro ?

```C
tmp = flb_sds_cat(sds_metrics, ....)
if (!tmp) {
    goto error;
}
else {
    sds_metric = tmp;
}